app/src/main/java/com/kickstarter/viewmodels/inputs/CreatorDashboardHeaderHolderViewModelInputs.java

```diff
@@ -0,0 +1,10 @@
+package com.kickstarter.viewmodels.inputs;
+
+import com.kickstarter.models.Project;
+import com.kickstarter.models.ProjectStats;
+
+
+public interface CreatorDashboardHeaderHolderViewModelInputs {
```

Comment:
ahh yes we're trying to move away from creating separate files for Inputs and Outputs which is why our ViewModels are interfaces, you can move these into the same file as `CreatorDashboardHeaderHolderViewModel`